javascript/javascript3/week1/weatherApp/script.js

```diff
@@ -0,0 +1,109 @@
+// @ts-nocheck
+window.addEventListener('load', () => {
+  let lat;
+  let long;
+  let geoBtn = document.querySelector('.geoBtn');
+  geoBtn.addEventListener('click', function getLocation() {
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition((position) => {
+        let lat = position.coords.latitude.toFixed(2); //to fixed is decimal
+        let long = position.coords.longitude.toFixed(2);
```

Comment:
lat and long variable are declared outside and inside this function, also if you want to do it inside it, you could use const instead of let :)